### PR TITLE
About page new design

### PR DESCRIPTION
**Pull request by theocampbell**
Merge only after merging sanity preview.

### CODE REVIEW

File: components/about-page-dropdown.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+              <Disclosure.Button as='a' href={`/about/${remainingTitles[1]?.slug}`}>
+                <div className='bg-white text-black-white-800 border-b border-l border-r border-green-500 h-10 flex items-center hover:text-green-500'>
+                  <span className='block pl-3 truncate'>{remainingTitles[1]?.title}</span>
+                </div>
+              </Disclosure.Button>
+
+              <Disclosure.Button as='a' href={`/about/${remainingTitles[2]?.slug}`}>
+                <div
+                  className={`${
+                    remainingTitles[2] === remainingTitles[-1] ? '' : 'rounded-b-md'
```
Comment (gurdenbatra) [inline components/about-page-dropdown.js:47]: hey am bit confused why we have `-1` index here?

File: components/about-page-dropdown.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+              </div>
+              <div className='w-1/12 px-5 h-full pr-5 bg-green-500 grid items-center justify-center rounded-r-md border border-green-500'>
+                <ChevronUpIcon
+                  className={`${open ? '' : 'rotate-180 transform'} h-5 w-5 text-white z-10`}
+                />
+              </div>
+            </Disclosure.Button>
+
+            <Disclosure.Panel>
+              <Disclosure.Button as='a' href={`/about/${remainingTitles[0]?.slug}`}>
```
Comment (gurdenbatra) [inline components/about-page-dropdown.js:33]: could we do` remainingTitles.map()` here?

File: components/about-page-preview.js [hunk 3]
```diff
@@ -3,36 +3,56 @@ import { PortableText } from '@portabletext/react';
 import OverNav from './over-nav';
 import { usePreview } from '../lib/sanity.preview';
 import { aboutPagePTComponents } from '../lib/portable-text/pt-components';
+import AboutPageDropdown from './about-page-dropdown';
+const margins = 'mx-8 md:mx-10 lg:ml-10 lg:mr-0 xl:ml-20 xl:mr-0 2xl:max-w-7xl 2xl:mx-auto';
```
Comment (gurdenbatra) [inline components/about-page-preview.js:7]: super small thing but maybe a newline after the imports :)

Comment (gurdenbatra) [reply components/about-page-preview.js:7]: maybe we can also make this into global-margin type thing for previews?

File: components/about-page.js [hunk 4]
```diff
@@ -2,28 +2,46 @@ import Link from 'next/link';
 import { PortableText } from '@portabletext/react';
 import OverNav from './over-nav';
 import { aboutPagePTComponents } from '../lib/portable-text/pt-components';
+import AboutPageDropdown from './about-page-dropdown';
+
+const margins = 'mx-8 md:mx-10 lg:ml-10 lg:mr-0 xl:ml-20 xl:mr-0 2xl:max-w-7xl 2xl:mx-auto';
```
Comment (gurdenbatra) [inline components/about-page.js:7]: same for this, any reason we can't use global margin as before?

File: lib/portable-text/portable-text-types.js [hunk 5]
```diff
@@ -1,7 +1,9 @@
 import Image from 'next/image';
 import { Disclosure } from '@headlessui/react';
 import CustomButton from '../../components/custom-button';
-import { ChevronUpIcon } from '@heroicons/react/outline';
+import { useNextSanityImage } from 'next-sanity-image';
+import { config } from '../config';
+import { IoMdArrowDropdownCircle } from 'react-icons/io';
```
Comment (gurdenbatra) [inline lib/portable-text/portable-text-types.js:6]: again super small thing but lets keep convention of imports from external packages over local ones

**Review by gurdenbatra — APPROVED** [review]

File: components/about-page-dropdown.js (hunk 2 above)
Comment (theocampbell) [reply components/about-page-dropdown.js:33]: I totally agree and I tried that but for some reason it was not working :/. When I would open and close the drop-down there would just be an empty div when using map.

File: components/about-page-dropdown.js (hunk 1 above)
Comment (theocampbell) [reply components/about-page-dropdown.js:47]: Just to add rounded corners on the last of the about pages. As in if index 2 is also -1 then it is the last.

File: components/about-page-preview.js (hunk 3 above)
Comment (theocampbell) [reply components/about-page-preview.js:7]: The design actually goes against the global margins. The over nav component goes to the very edge of the page for LG and XL screen sizes.

File: lib/portable-text/portable-text-types.js (hunk 5 above)
Comment (theocampbell) [reply lib/portable-text/portable-text-types.js:6]: Oops! Will fix that up before merging

File: components/about-page-dropdown.js (hunk 1 above)
Comment (gurdenbatra) [reply components/about-page-dropdown.js:47]: ohhhh

File: components/about-page-preview.js (hunk 3 above)
Comment (gurdenbatra) [reply components/about-page-preview.js:7]: maybe we create an alternate global margin then?

File: components/about-page-dropdown.js (hunk 2 above)
Comment (gurdenbatra) [reply components/about-page-dropdown.js:33]: strange!

**Pull request merged**